### PR TITLE
feat(DB/lang): Import German quest_request_items Texts from GTDB (Part 3/4)

### PR DESCRIPTION
**Pull request by LukasVolgger**
Part 3/4 to import the german quest_request_items texts from GTDB.
None of these IDs previously existed in German, so they were displayed in English.

## CHANGES PROPOSED:
-  Import the third part ~ 750 of a total of ~ 3000 German quest texts for table `quest_request_items_locale` from GTDB 

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
- Corrected each string with the punctuation marks from the `quest_request_items` table
- Checked for spelling and grammatical errors and corrected 
- Imported on win 10 pro x64 with HeidiSQL without errors
- Checked a few quests ingame

## HOW TO TEST THE CHANGES:

To test changes ingame you need deDE client. Since I read almost every string and compared the correctness using the `quest_request_items` table, it is sufficient to just test the SQL. 

- Import SQL file and check that no error occurs
- Can be merged after approval

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1582975268828510900.sql [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+(9854,'deDE','Habt Ihr schon den Frieden gefunden, der einen überkommt, wenn man seinen Platz in der Natur gefunden hat? Seid Ihr Jäger oder Gejagter?',18019),
+(9855,'deDE','Habt Ihr inzwischen gelernt, Euren Herzschlag zu zügeln? Ruhig und geduldig wie ein stiller See zu sein und auf den richtigen Moment zu warten, um die Beute zu erlegen?',18019),
+(9857,'deDE','Ich bleibe hier einfach noch ein bisschen liegen. Es geht mir sicher bald besser.',18019),
+(9858,'deDE','Ich nehme nicht an...<hust>...dass Ihr Euch schon...<hust>..um die Dornzehrer gekümmert habt?',18019),
+(9862,'deDE','Der Luftgeist meint, dass die Finsterblut in Sonnenwind nach etwas suchen...',18019),
+(9865,'deDE','Für jeden meiner Brüder, den sie umgebracht haben, müssen zehn von ihnen sterben!',18019),
+(9878,'deDE','Die Finsterblut sind ein ruchloser Stamm der Zerschlagenen, $N. Sie mögen vielleicht aussehen wie Kurenai, aber tatsächlich sind sie wilde, mordende Tiere.',18019),
+(9894,'deDE','Habt Ihr Lord Klaq gefunden und vernichtet?',18019),
+(9895,'deDE','Habt Ihr Sumpfschläger gefunden?',18019),
+(9900,'deDE','Ihr seid zurück, $R. Ist Gava\'xi tot?',18019),
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1582975268828510900.sql]: better using `''` instead of `\'` to escape SQL

Comment (Volgger) [reply data/sql/updates/pending_db_world/rev_1582975268828510900.sql]: Updated - it's the same in my other PRs because it's a HeidiSQL outpot.

**Comment by Volgger** [comment]
Instructions for testing updated.

**Comment by Kaev** [comment]
I didn't test ingame but the SQL looks fine to me. Valid translation aswell.

**Comment by Volgger** [comment]
Thx @Kaev !

**Pull request merged**